deploy/Chart/templates/de/deployment.yaml

```diff
@@ -53,6 +63,10 @@ spec:
           value: "true"
         - name: RADIUSBACKENDURL
           value: https://ucp.radius-system:443/apis/api.ucp.dev/v1alpha3
+        {{- if .Values.global.caCert.filename }}
+        - name: SSL_CERT_DIRS
+          value: {{ .Values.global.caCert.mountPath }}
```

Comment:
Do we need changes to the DE to use this?

Reply:
Good catch. This works only for Go. Let me find the solution to set intermediate CA for Dotnet Core.

Reply:
Looks like dotnet runtime uses the same env vars - https://github.com/dotnet/runtime/issues/27658#issuecomment-431243158

Since DE still calls ARM and AAD endpoints, we need this one.

Reply:
👍